bundles/binding/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/handling/ComfoAirConnector.java

```diff
@@ -423,4 +437,15 @@ private String dumpData(byte[] data) {
         }
         return sb.toString();
     }
+
+    private class CPUWorkaroundThread implements SerialPortEventListener {
+        @Override
+        public void serialEvent(SerialPortEvent event) {
+            try {
+                logger.trace("RXTX library CPU load workaround, sleep forever");
+                Thread.sleep(Long.MAX_VALUE);
+            } catch (InterruptedException e) {
+            }
+        }
+    }
```

Comment:
@csowada do you think my issue [https://github.com/openhab/openhab2-addons/issues/1777](url)  could be related to the problem you are fixing here?  my issue is around Powermax binding, but it sounds similar to me?